static/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -337,7 +337,9 @@ export default class AbstractExternalIssueForm<
 
     return (
       <Fragment>
-        <Header closeButton>{this.getTitle()}</Header>
+        <Header closeButton>
+          <h4>{this.getTitle()}</h4>
```

Comment:
you know. When i refactored all the headers, I tried to REMOVE all these h4s, in hopes of just changing the standard header size.

How hard do you think that would be?

Reply:
you'd have to fight @robinrendle

Reply:
if we follow this pattern of always having `<h4>` in the header of modals, we must create a reusable component for it